client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -593,6 +594,9 @@ export class CheckoutThankYou extends Component<
 					<PlanOnlyThankYou
 						primaryPurchase={ purchases[ 0 ] }
 						isEmailVerified={ this.props.isEmailVerified }
+						errorNotice={ this.props.errorNotice }
+						removeNotice={ this.props.removeNotice }
+						successNotice={ this.props.successNotice }
```

Comment:
@nightnei can probably better explain this, my understanding is that, these functions need to be passed from the main component to `<PlanOnlyThankYou>`, otherwise the notices won't work. These functions don't actually trigger the rendering of success/error notices, but instead, they generate objects that are fed to redux events that trigger the rendering of notices. This is the reason why the notices aren't showing up when testing the original PR https://github.com/Automattic/wp-calypso/pull/88152.

Reply:
1) I would put it inside the `PlanOnlyThankYou` component, I don't see necessity to provide it from outside. Also, it will make code cleaner w/o these 3 props.
2) Why it doesn't work in the initial PR - since in the initial PR we are using just simple functions which are not wrapped with redux-connect: https://github.com/Automattic/wp-calypso/pull/88256/files/c8014a0dc950ca5f1b4012c5d4416f06fd75524a#diff-daac408bb350333e9fb8ca7c03233e75f03656e02a45c152e574227e6d253ee7R997-R999

Reply:
Thanks @nightnei for walking me through the changes! https://github.com/Automattic/wp-calypso/pull/88256/commits/2c305bdcf9c432a900f268a994e35c07d4373db3